### DISCUSSION TITLE
Thank you, developer!

### DISCUSSION THREAD

**Discussion by Sanek-Dev** [xCollateral/VulkanMod#171] [General]
Эта дискуссия не имеет смысла, но я просто хочу поблагодарить разработчика за его старания над модом!
// Google translator be like
// **This discussion is meaningless, but I just want to thank the developer for the efforts on the mod!**

**Comment by AnemoFlower** [comment]
希望这个 Mod 能流畅运行光影！（即使不能直接使用现有的）

I'm looking forward to using this Mod to add shaders and play smoothly! ( Even if you can't use the existing light and shadow directly)

**Reply by Yuyuyuyu-an** [reply]
现在已经有Baryl了！
Now Beryl is available.
